lib/createReturnedResults.mjs

```diff
@@ -0,0 +1,33 @@
+import createPartialStylelintResult from './createPartialStylelintResult.mjs';
+import getConfigForFile from './getConfigForFile.mjs';
+
+/**
+ * @param {import('stylelint').InternalApi} stylelint
+ * @param {import('stylelint').PostcssResult} [postcssResult]
+ * @param {string} [filePath]
+ * @param {import('stylelint').CssSyntaxError} [cssSyntaxError]
+ * @return {Promise<import('stylelint').LintResult>}
+ */
+export default async function createStylelintResult(
+	stylelint,
+	postcssResult,
+	filePath,
+	cssSyntaxError,
+) {
+	let stylelintResult = createPartialStylelintResult(postcssResult, cssSyntaxError);
```

Comment:
[suggest] I want to avoid one more nest for the experimental feature. Does it make sense to modify `createPartialStylelintResult()` instead of adding the `createStylelintResult()` module?

Reply:
Agree. I'm keeping the original createPartialStylelintResult, and adding a new function underneath:
```js
  const stylelintResult = createPartialStylelintResult(postcssResult);
  await postProcessStylelintResult(stylelint, stylelintResult, absoluteFilepath);
```
https://github.com/YuanboXue-Amber/stylelint/blob/985bdf1a7f7cd7999867f97153ca4cc440dbb643/lib/standalone.mjs#L250-L252